querybook/server/lib/data_doc/dag_exporter/demo_dag_exporter.py

```diff
@@ -31,9 +59,41 @@ def dag_exporter_meta(self):
 
     def export(self, nodes, edges, meta):
         try:
+            export_dag = ""
+
+            for node in nodes:
+                node_section = '''
+    query = """{}"""
+    templated_query = dedent(query)
+
+    node{} = BashOperator(
+        task_id="cell_{}", depends_on_past=False, bash_command=templated_query,
+    )
+                '''.format(
+                    node["data"]["query"], node["id"], node["id"]
+                )
+                export_dag = export_dag + node_section
+
+            source_to_target = {}
+            for edge in edges:
+                source_id = str(edge["source"])
```

Comment:
how come source casted to str but target is kept as is?

Reply:
target doesnt become a key?